.gitignore

```diff
@@ -1,13 +1,32 @@
 # Cache files
 __pycache__/
 *cache*
+<<<<<<< HEAD
```

Comment:
This looks like an unresolved merge conflict?

Reply:
Oops, will take care of that